config/development-amo.js

```diff
@@ -1,5 +1,10 @@
 module.exports = {
   apiHost: 'http://localhost:3000',
+  // Setting this to false returns add-ons that are not compatible but means
+  // developers can pull from a much larger dataset on the local/-dev/-stage
+  // servers. Set this to true to only get compatible add-ons (this is what
+  // happens in production) but get a lot fewer add-ons in search results.
+  checkForVersionCompatiblityInAPIRequests: false,
```

Comment:
Is this absolutely necessary? I would rather see this as a config option that we explicitly set in our local servers only. I am just worried that dev will stray too far from production by disabling this feature.

Reply:
I thought the `dev` config was for the `dev` server and this config would only affect local development. If I'm wrong on that this wasn't intended and yeah I only want this on local dev.

Reply:
To clarify: I wanted it enabled by default because otherwise the data set is near useless for development.

I agree: I'd rather us clean up the data on the servers, but I know that's a lot to ask of the server devs so until then I'd rather fix it this way for anyone doing front-end development and not require more docs/manual work to set the config value.

Reply:
Ah, whoops, I was mistaken. Yes, this only applies to local development which I was asking for so carry on 🌴 🌞